fineract-provider/src/main/java/org/apache/fineract/portfolio/loanproduct/service/LoanProductWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -150,6 +155,16 @@ public CommandProcessingResult createLoanProduct(final JsonCommand command) {
 
     }
 
+    private void validateShortName(JsonCommand command) {
```

Comment:
We dont need this. 
Please check why the LoanProductWritePlatformServiceJpaRepositoryImpl.handleDataIntegrityIssues() method was not handling this properly.